tests/api/test_ratelimitinfo.py

```diff
@@ -0,0 +1,45 @@
+from datetime import datetime, timezone, timedelta
+
+from api import RateLimitInfo
```

Comment:
```suggestion
from libtado.api import RateLimitInfo
```

This will lead the `pytest` command successfully.

```bash
poetry run pytest -sv tests/
```

Reply:
Fixed. I had ran the tests with IntelliJ, which apparently does something different because it worked with just `api`.

Now the tests also pass with Poetry. (and of course setting `TADO_CREDENTIALS_FILE`)